### PR TITLE
LPS-117031 Updates lodash to 4.17.19

### PR DESCRIPTION
**Pull request by diegonvs**
### How I fixed It

Running `yarn why lodash`, I noticed We're using a unsafe version inside lodash-web module and inside `liferay-theme-tasks/gulp-replace-tasks`. See https://gist.github.com/diegonvs/65c9a153008e8e88565e1ced7258f72e

Updating lodash version from lodash-web and rerrunning again `yarn why lodash`: https://gist.github.com/diegonvs/2c7ce577cd50dac60cd658057533ff0b

The most part of usages were fixed but inside `liferay-theme-tasks/gulp-replace-tasks` it won't because: https://github.com/liferay/liferay-js-themes-toolkit/issues/199

### CODE REVIEW

**Comment by diegonvs** [comment]
ci:test:sf

**Comment by diegonvs** [comment]
ci:test:relevant

**Comment by jbalsas** [comment]
LGTM!

File: modules/yarn.lock [hunk 1]
```diff
@@ -7270,7 +7270,7 @@ discontinuous-range@1.0.0:
   resolved "https://registry.yarnpkg.com/discontinuous-range/-/discontinuous-range-1.0.0.tgz#e38331f0844bba49b9a9cb71c771585aab1bc65a"
   integrity sha1-44Mx8IRLukm5qctxx3FYWqsbxlo=
 
-dnd-core@^10.0.1:
+dnd-core@10.0.1, dnd-core@^10.0.1:
```
Comment (jbalsas) [inline modules/yarn.lock:7273]: makes me wonder where this comes from... 🤔

**Comment by jbalsas** [comment]
Going to forward since this is a "security" vulnerability and since we don't use lodash, impact should be minimal.

**Comment by jbalsas** [comment]
ci:forward

File: modules/yarn.lock (hunk 1 above)
Comment (wincent) [reply modules/yarn.lock:7273]: I've seen this particular line flipping back and forth. I am suspicious that it is very sensitive to local state on the developer machine.

It [conflicted](https://github.com/brianchandotcom/liferay-portal/pull/91298) anyway with [my change](https://github.com/brianchandotcom/liferay-portal/pull/91296). For that reason, unless I think I'm the only one touching the `yarn.lock` file, I keep stuff like this (ie. stuff that doesn't change the actual resolution) out of the PR.

Comment (jbalsas) [reply modules/yarn.lock:7273]: > It conflicted anyway with my change. For that reason, unless I think I'm the only one touching the yarn.lock file, I keep stuff like this (ie. stuff that doesn't change the actual resolution) out of the PR.

😂 Yeah... less is always more, @diegonvs! 😉

**Pull request closed without merging**